ee/tables/execparsers/key_value/key_value.go

```diff
@@ -0,0 +1,41 @@
+//go:build darwin || linux
+// +build darwin linux
+
+package key_value
+
+import (
+	"io"
+)
+
+// parser implements the parser interface for key-value data
+type parser struct {
+	kvDelimiter string
+}
+
+// Parser is the singleton instance of the parser
+var Parser = New()
```

Comment:
Do we want a singleton instance to be available here? Wondering if we might accidentally use this in the future in a couple places and not realize when calling `SetDelimiter` that we've updated the delimiter for both tables